sdcm/sct_config.py

```diff
@@ -1151,8 +1144,8 @@ def __init__(self):
                             self[key] += " {}".format(value)
 
         # 5) handle scylla_version if exists
-        scylla_version = self.get('scylla_version', None)
-        scylla_linux_distro = self.get('scylla_linux_distro', '')
+        scylla_version = self.get('scylla_version')
+        scylla_linux_distro = self.get('scylla_linux_distro') or ''
```

Comment:
remove '', the default should be in the defaults yaml

Reply:
done